packages/fast-components-class-name-contracts-base/src/breadcrumb-item/index.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * The class name contract for the breadcrumb-item component
+ */
+export interface BreadcrumbItemClassNameContract {
+    breadcrumbItem?: string;
+    breadcrumbItem__current?: string;
+    breadcrumbItem__hypertext?: string;

```

Comment:
I don't think this is always going to be hypertext. We should be allowing "children", so I would suggest a more generic approach.